src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/VariableSizedWrapGrid.cs

```diff
@@ -145,7 +147,7 @@ public  double ItemHeight
 		Windows.UI.Xaml.DependencyProperty.RegisterAttached(
 			"RowSpan", typeof(int), 
 			typeof(global::Windows.UI.Xaml.Controls.VariableSizedWrapGrid), 
-			new Windows.UI.Xaml.FrameworkPropertyMetadata(default(int)));
+			new Windows.UI.Xaml.FrameworkPropertyMetadata(Boxes.DefaultBox<int>.Value));
```

Comment:
I'd tend to say that if we know what the default is, let's use the right one explicitly, it will avoid a generic static resolution indirection.

Reply:
`object Box(int value)` currently uses `DefaultBox<int>` anyway.

Should I actually remove `DefaultBox<T>` completely and only have boxes for known specific types?

Reply:
As we're only going to be using this internally, I think it makes sense, yes.